### PR TITLE
Decouple the nearby view from the nearby activity

### PR DESCRIPTION
**Pull request by rogernkosi**

### CODE REVIEW

**Comment by rogernkosi** [comment]
[Decouple Nearby Activity to Fragment](https://trello.com/c/mJLQmKYw/23-decouple-nearby-activity-to-fragment)

**Comment by rogernkosi** [comment]
[Android Fragment Navigation without Graph used in latest Android Components](https://trello.com/c/yVgTwQZq/19-android-fragment-navigation-without-graph-used-in-latest-android-components)

**Comment by rogernkosi** [comment]
[Use Nearby Activity as navigation activity - removing the timetable activity](https://trello.com/c/rOeMkUuX/20-use-nearby-activity-as-navigation-activity-removing-the-timetable-activity)

File: app/build.gradle [hunk 1]
```diff
@@ -6,6 +6,9 @@ apply plugin: 'kotlin-android-extensions'
 
 apply plugin: 'kotlin-kapt'
 
+apply plugin: 'androidx.navigation.safeargs'
```
Comment (N2Android) [inline app/build.gradle]: - Remove unused dependency

Comment (N2Android) [reply app/build.gradle]: Already done on different pull request.

**Pull request merged**